src/components/Dropmenu/index.tsx

```diff
@@ -0,0 +1,57 @@
+import clsx from 'clsx';
+import React, { useState } from 'react';
+import { ReactComponent as ArrowDownIcon } from 'assets/revamp-icon/arrow-down.svg';
+import { MenuItem } from 'components/NavHeader';
+
+type DropmenuProps = {
+  children: React.ReactNode;
+  menu?: MenuItem[];
+};
+
+const Dropdown = (props: DropmenuProps) => {
+  const { children, menu = [] } = props;
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsOpen(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <div
+      className="relative inline-block"
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+    >
+      <div className="inline-flex items-center justify-center">
+        {children}
+        {menu?.length > 0 && (
+          <ArrowDownIcon
+            className={clsx('transition-all duration-300 ease-in-out', isOpen ? 'rotate-180' : '')}
+          />
+        )}
+      </div>
+
+      {isOpen ? (
```

Comment:
you do not need to make it null, just do isOpen &&

Reply:
Updated